doc/modules/model_evaluation.rst

```diff
@@ -2241,8 +2266,11 @@ predicted by the model, through the proportion of explained variance.
 As such variance is dataset dependent, R² may not be meaningfully comparable
 across different datasets. Best possible score is 1.0 and it can be negative
 (because the model can be arbitrarily worse). A constant model that always
-predicts the expected value of y, disregarding the input features, would get a
-R² score of 0.0.
+predicts the expected (average) value of y, disregarding the input features,
+would get a :math:`R^2` score of 0.0.
+
+Note: when the prediction residuals have zero mean (perfectly unbiased
+model), the :math:`R^2` score and the :ref:`explained_variance_score` are identical.
```

Comment:
```suggestion
Note: when the prediction residuals have zero mean (an unconditionally calibrated/unbiased
model), the :math:`R^2` score and the :ref:`explained_variance_score` are identical.
```

Reply:
Although I trust that this is accurate for some statistics community, I am not very fond of this wording as it makes it much harder to understand for a "standard" user (e.g. Machine Learning community or broad data scientist).

Reply:
The point is that a model with zero mean residual is by far not "perfectly unbiased". For instance, the mean residual grouped by some (categorical) feature could be very unbiased.

Reply:
Fair point. What about simply "(unbiased model)" or even removing the parenthesis ? That would be simpler to read

Reply:
@glemaitre What do you think?

I would be either correct (my suggestion) or remove the parenthesis.

Reply:
removing parenthesis is good